Detectors/MUON/MCH/Clustering/include/MCHClustering/dataStructure.h

```diff
@@ -0,0 +1,74 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef _DATASTRUCT_H
+#define _DATASTRUCT_H
```

Comment:
Please make sure you use the guard naming as proposed by the naming convention ( see https://rawgit.com/AliceO2Group/CodingGuidelines/master/coding_guidelines.html?showone=The__define_Guard#The__define_Guard).

Reply:
Thank you for this mistake and the documentation.
A new re-factorized version is in preparation. This PR is only to test numerically 
the algorithm (the original clustering is unmodified).